cms/static/cms/js/modules/cms.modal.js

```diff
@@ -373,6 +373,7 @@ var CMS = window.CMS || {};
                 // add esc close event
                 this.ui.body.off('keydown.cms.close').on('keydown.cms.close', function (e) {
                     if (e.keyCode === CMS.KEYS.ESC) {
+                        that.options.onClose = null;
```

Comment:
what should it ask for? The only check happens, if you create the first plugin and thats still the case. Every close event (pressing on cancel, esc or x) cancels the reload or "onClose"